teams/chef-foundation.md

```diff
@@ -0,0 +1,55 @@
+# Chef Foundation Team
+
+The Chef Foundation team is responsible for common libraries and applications used to support projects such as Chef and InSpec.
+
+### Project Owners
+
+Clinton Wolfe
+  - Github: [clintoncwolfe](https://github.com/clintoncwolfe)
+  - Slack: @btm
+
+Lamont Granquist
+  - Github: [lamont-granquist](https://github.com/lamont-granquist)
+  - Slack: @lamont
+
+Marc Paradise
+  - Github: [marcparadise](https://github.com/marcparadise)
+  - Slack: @mparadise
+
+### Project Approvers
+
+Miah Johnson
+  - Github: [miah](https://github.com/miah)
+  - Slack: @miah
+
+Tim Smith
+  - Github: [tas50](https://github.com/tas50)
+  - Slack: @tas50
+
+Tyler Ball
+  - Github: [tyler-ball](https://github.com/tyler-ball)
+  - Slack:
+
+### Project Reviewers
+
+## Project Repositories
+
+[appbundle-update](https://github.com/chef/appbundle-update)
+[appbundler](https://github.com/chef/appbundler)
+[chef-telemetry](https://github.com/chef/chef-telemetry)
+[chefstyle](https://github.com/chef/chefstyle)
+[ffi-libarchive](https://github.com/chef/ffi-libarchive)
+[ffi-yajl](https://github.com/chef/ffi-yajl)
+[kitchen-appbundle-updater](https://github.com/chef/kitchen-appbundle-updater)
+[libyajl2-gem](https://github.com/chef/libyajl2-gem)
+[mixlib-archive](https://github.com/chef/mixlib-archive)
+[mixlib-authentication](https://github.com/chef/mixlib-authentication)
+[mixlib-cli](https://github.com/chef/mixlib-cli)
+[mixlib-config](https://github.com/chef/mixlib-config)
+[mixlib-install](https://github.com/chef/mixlib-install)
+[mixlib-log](https://github.com/chef/mixlib-log)
+[mixlib-shellout](https://github.com/chef/mixlib-shellout)
+[mixlib-versioning](https://github.com/chef/mixlib-versioning)
+[omnibus-software](https://github.com/chef/omnibus-software)
+[omnibus](https://github.com/chef/omnibus)
```

Comment:
With omnibus repos in here, maybe some representation from Chef's Release Engineering? @scotthain?

Maybe also include `[omnibus-toolchain](https://github.com/chef/omnibus-toolchain)`

Reply:
I also volunteer as a Reviewer having had my hands in `omnibus*`, `chef-telemetry`, and the occasional `mixlib`.

Reply:
yep @scotthain should be in here if he wants.

Reply:
I'm fine with  being an approver/reviewer